code/modules/mob/living/carbon/human/life.dm

```diff
@@ -28,25 +28,26 @@
 	if (QDELETED(src))
 		return 0
 
-	if(.) //not dead
-		handle_active_genes()
+	if(!IsInStasis())
+		if(.) //not dead
+			handle_active_genes()
```

Comment:
This may kill you due to calling `on_life()` on all human mutations.
https://github.com/tgstation/tgstation/blob/5b89dea70f84fe72a653c4d65fbf056206764fc3/code/modules/mob/living/carbon/human/life.dm#L313-L314